### DISCUSSION TITLE
XAML User Control vs Uno User Control?

### DISCUSSION THREAD

**Discussion by TonyK-FmGroup** [unoplatform/uno#8536] [Q&A] [answered]
Hello again!
When adding a new user control to the shared project (VS 2022), is there any difference in using the "User Control" template under the 'XAML' catgeory and the "Custom Control (Uno Platform UWP)", will the latter only work with UWP?

thanks in advance 
Tony

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question! Both templates are behaving the same. The Uno Platform template are present for the IDE cases where the UWP templates don't appear (sometimes in Shared projects, depending on visual studio versions). 

The mention for UWP only means it's using the Windows.UI.Xaml namespace (in comparison to Microsoft.UI.Xaml found in WinUI 3/WinApp SDK).
